plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -1679,12 +1679,12 @@ private void testDBRefLikeDocument(Document document1, Document document2, Strin
         assertThat(query("SELECT creator.id FROM test." + tableName))
                 .skippingTypesCheck()
                 .matches("VALUES (%1$s), (%1$s)".formatted(expectedValue))
-                .isNotFullyPushedDown(ProjectNode.class);
+                .isFullyPushedDown();
 
-        assertThat(query("SELECT creator.databasename, creator.collectionname, creator.id FROM test." + tableName))
+        assertThat(query("SELECT creator.databaseName, creator.collectionName, creator.id FROM test." + tableName))
```

Comment:
Is this expected to be case insensitive? If so I can add support for it but from what I understand Trino is case-sensitive.

Reply:
It's intentional. Please revert the change.